packages/synapse-core/tools/endorse-sp.js

```diff
@@ -1,19 +1,28 @@
+import EthModule from '@ledgerhq/hw-app-eth'
+import TransportNodeHidModule from '@ledgerhq/hw-transport-node-hid'
 import { createWalletClient, http } from 'viem'
-import { privateKeyToAccount } from 'viem/accounts'
+import { privateKeyToAccount, toAccount } from 'viem/accounts'
+
+const TransportNodeHid = TransportNodeHidModule.default || TransportNodeHidModule
+const Eth = EthModule.default || EthModule
+
 import { getChain } from '../src/chains.ts'
 import { signEndorsement } from '../src/utils/cert.ts'
 
 function printUsageAndExit() {
-  console.error('Usage: PRIVATE_KEY=0x... node utils/endorse-sp.js providerId...')
+  console.error('Usage: PRIVATE_KEY=0x... node tools/endorse-sp.js providerId...')
+  console.error('   or: USE_LEDGER=true node tools/endorse-sp.js providerId...')
   process.exit(1)
 }
 
 const PRIVATE_KEY = process.env.PRIVATE_KEY
+const USE_LEDGER = process.env.USE_LEDGER === 'true'
+const LEDGER_PATH = process.env.LEDGER_PATH || "m/44'/60'/0'/0/0"
```

Comment:
```suggestion
const LEDGER_PATH = process.env.LEDGER_PATH ?? "m/44'/60'/0'/0/0"
```

see https://github.com/FilOzone/synapse-sdk/pull/443#discussion_r2561135276 - `??` is a newer operator that avoids the vagaries of JS's "falsy" values, where various strings can evaluate to `false` which makes it easier to fall through to the default value here. But `??` gives us strictness where if someone provides a `LEDGER_PATH` that happens to be falsy too, like `"0"`, or even `""`, it should end up failing rather than silently passing and using the default path which may be a surprise.